website/manual.md

```diff
@@ -312,57 +312,51 @@ file_server --reload
 
 ### Run subprocess
 
-```
+[API Reference](https://deno.land/typedoc/index.html#run)
+
+Example:
+
+```ts
 const p = Deno.run({
-    args: ["deno", "--allow-read", "https://deno.land/x/examples/cat.ts", "README.md"],
+  args: ["echo", "hello"]
 });
 
 // start subprocess
 await p.status();
 ```
 
-When this program is started, the user is prompted for permission to run
-subprocess:
-
-```
-> deno https://deno.land/x/examples/subprocess_simple.ts
-⚠️  Deno requests access to run a subprocess. Grant? [yN] y
-```
-
-For security reasons, deno does not allow programs to run subprocess without
-explicit permission. To avoid the console prompt, use a command-line flag:
+Run it:
 
 ```
-> deno https://deno.land/x/examples/subprocess_simple.ts --allow-run
+> deno --allow-run https://deno.land/x/examples/subprocess_simple.ts
```

Comment:
Let's just use a normal filename for now - just to make sure people understand that things can be run from the local disk too...